packages/checkout-com/src/useCkoCard.ts

```diff
@@ -113,6 +113,8 @@ const useCkoCard = (selectedPaymentMethod: Ref<CkoPaymentType>) => {
     selectedPaymentMethod.value = CkoPaymentType.SAVED_CARD;
   };
 
+  onUnmounted(removeTransactionToken);
```

Comment:
Can you clarify when this gets triggered?

Reply:
You were right. I made changes. Since now you will have to append this logic manually in your app - I added bullet about this to the readme